### ISSUE TITLE
[Site]: support structured schema (JSON-LD)

### ISSUE THREAD

**Issue by juzhiyuan** [iresty/iresty.github.io#980]
### Current State

**Description of the current state/issue**: For each page under the Apache APISIX Website, we don't provide a structured schema for search engines, so the search engine has to try to understand each page's content meaning. Structured Schema is an Advanced SEO solution, please refer to the following links for more information.

![image](https://user-images.githubusercontent.com/2106987/160530278-cc064c3e-9a4d-4eaa-b23f-331b710d1caf.png)

1. https://moz.com/blog/json-ld-for-beginners
2. https://developers.google.com/search/docs/advanced/structured-data/intro-structured-data
3. https://stackoverflow.com/questions/38670851/whats-a-script-type-application-ldjsonjsonobj-script-in-a-head-sec/38745339#38745339
4. https://support.google.com/knowledgepanel/answer/9163198?hl=zh-Hans&ref_topic=9164489
5. https://developers.google.com/search/docs/advanced/guidelines/get-started

### Desired State

Validator Tools: https://validator.schema.org/

**Comment by SkyeYoung** [comment]
If we need a good type prompt, we can use https://www.npmjs.com/package/schema-dts.

**Comment by nfrankel** [comment]
I'm interested to contribute on this one. It can be a quick win.

**Comment by juzhiyuan** [comment]
Hi, Nicolas, Thanks!!

**Comment by nfrankel** [comment]
I've checked in more detail and it's not as quick a win as I thought.

The number of items is pretty limited. I believe we can use [`How-To`](https://developers.google.com/search/docs/advanced/structured-data/how-to) and [`Article`](https://developers.google.com/search/docs/advanced/structured-data/article) quite easily. Others, such as `Learning Video` and `Software App` are a bit harder to push in.

Hence, I think I could start with one article from the blog and see what I can make out of it.

**Comment by pottekkat** [comment]
@juzhiyuan @nfrankel @SkyeYoung we have to work on this to improve discoverability in Google. From experience, adding schema to the site can help in getting a "Knowledge Panel" in Google search. @SkyeYoung With your help, I can probably look into what we can do and take this forward.

**Comment by nfrankel** [comment]
Yes, I wanted to work on that months ago... and did nothing. Feel free to own it
